git/functions/gdel

```diff
@@ -3,7 +3,14 @@
 # delete given branch locally & remotely
 
 [[ -z $1 ]] && {
-  echo 'No branch given, aborting'
+  echo 'error: the following arguments were not provided'
+  echo ''
+  echo '  BRANCH-NAME'
+  echo ''
+  echo 'USAGE:'
+  echo ''
+  echo '    gdel BRANCH-NAME'
```

Comment:
to make it even more accurate could add `...` to indicate that multiple `BRANCH-NAME`s can be given - `  gdel BRANCH-NAME...`

Reply:
done